example_payloads/variant_calling.payload.json

```diff
@@ -0,0 +1,124 @@
+{
+  "studyId" : "TEST-PR",
+  "analysisType" : {
+    "name" : "variant_calling"
```

Comment:
@lindaxiang - As discussed, can we rename this to something more unique for EUCANCan?  I don't think we should call it federated_variant_calling, because it may be confusing.  Can we rename to variant_calling_eucancan?